Makefile

```diff
@@ -101,7 +101,7 @@ vet: ## Run go vet against code.
 ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: manifests generate fmt vet test-fast ## Run tests.
 
-TEST_PACKAGES := ./...
+TEST_PACKAGES := ./controllers
```

Comment:
why there is a need to narrow the packages ?

Reply:
oops....fixing :-)